applications/GeoMechanicsApplication/tests/test_helper.py

```diff
@@ -419,6 +420,97 @@ def find_closest_index_greater_than_value(input_list, value):
     return None
 
 
+def are_values_almost_equal(expected: Any, actual: Any, abs_tolerance: float = 1e-7) -> bool:
+    """
+    Checks whether two values are almost equal.
+
+    Args:
+        - expected (Any): Expected value.
+        - actual (Any): Actual value.
+
+    Returns:
+        - True if the values are almost equal, False otherwise.
+
+    """
+    # check if the value is a dictionary and check the dictionary
+    if isinstance(expected, dict):
+        # check if the value is a dictionary and check the dictionary
+        if not are_dictionaries_almost_equal(expected, actual):
+            return False
```

Comment:
To reduce the number of branches, I would opt for an early `return`:
```suggestion
        return are_dictionaries_almost_equal(expected, actual)
```

Also, since the function name is very descriptive the comment doesn't add anything that the code already tells.